util/chunk/row_container.go

```diff
@@ -211,6 +215,11 @@ func (c *RowContainer) GetDiskTracker() *disk.Tracker {
 
 // Close close the RowContainer
 func (c *RowContainer) Close() (err error) {
+	if c.actionSpill != nil {
+		// Set status to spilledYet to avoid spilling.
+		c.actionSpill.setStatus(spilledYet)
+		c.actionSpill.cond.Broadcast()
+	}
```

Comment:
Should it be in `c.m.RLock()`, or the spilling status may be changed back to `spilling` at line 75

Reply:
Addressed.